### PR TITLE
feat(shadcn-ui): move shadcn-ui to shadcn

### LINKED ISSUE #309: Error when running `nx run ui-lib:add-component ....` due to 'shadcn-ui' not being recognized
## Description

When trying to add a component using the command `nx run ui-lib:add-component card`, I encountered an error indicating that `shadcn-ui` is not recognized as a command.

It worked yesterday, but today it doesn't! I think it's due to the shadcn/cli upgrade!

### Command Run

```bash
nx run ui-lib:add-component card
```

### Output/Error Message

```bash
Running:
npx shadcn-ui@latest add card --path=src --cwd=ui-lib
'shadcn-ui' is not recognized as an internal or external command,
operable program or batch file.

—————————————————————————————————————————————————————————————————————————————————————————————————————————————— 

 NX   Ran target add-component for project ui-lib (3s)

      With additional flags:
        card

   ×  1/1 failed
   √  0/1 succeeded [0 read from cache]
```

### Expected Behavior

The command should successfully add the `card` component to the project without any errors.

### Steps to Reproduce

1. Run `nx run ui-lib:add-component card`.
2. Observe the error related to `shadcn-ui`.

### Additional Context

If this issue is specific to my environment or setup, please let me know if there are any dependencies or steps I might be missing.

### PR DESCRIPTION
**Pull request by bzp2010**
Fix #309 #340

Move `shadcn-ui` CLI to `shadcn` CLI for add components.

### CODE REVIEW

**Comment by bzp2010** [comment]
About `add` command:

```
➜  yap git:(test) ✗ TS_NODE_PROJECT=tsconfig.base.json pnpm dlx shadcn@latest add sidebar
✔ Checking registry.
✔ Updating packages/shadcn-ui-utils/src/tailwind.config.ts
✔ Updating packages/shadcn-ui-utils/src/global.css
✔ Installing dependencies.
✔ The file sidebar.tsx already exists. Would you like to overwrite? … yes
✔ The file button.tsx already exists. Would you like to overwrite? … yes
✔ The file separator.tsx already exists. Would you like to overwrite? … yes
✔ The file sheet.tsx already exists. Would you like to overwrite? … yes
✔ The file tooltip.tsx already exists. Would you like to overwrite? … yes
✔ The file input.tsx already exists. Would you like to overwrite? … yes
✔ The file use-mobile.tsx already exists. Would you like to overwrite? … yes
✔ The file skeleton.tsx already exists. Would you like to overwrite? … yes
ℹ Updated 8 files:
  - packages/shadcn-ui/src/ui/sidebar.tsx
  - packages/shadcn-ui/src/ui/button.tsx
  - packages/shadcn-ui/src/ui/separator.tsx
  - packages/shadcn-ui/src/ui/sheet.tsx
  - packages/shadcn-ui/src/ui/tooltip.tsx
  - packages/shadcn-ui/src/ui/input.tsx
  - packages/shadcn-ui/src/hooks/use-mobile.tsx
  - packages/shadcn-ui/src/ui/skeleton.tsx

➜  yap git:(test) ✗ TS_NODE_PROJECT=tsconfig.base.json pnpm dlx shadcn@latest add sidebar --overwrite
✔ Checking registry.
✔ Updating packages/shadcn-ui-utils/src/tailwind.config.ts
✔ Updating packages/shadcn-ui-utils/src/global.css
✔ Installing dependencies.
ℹ Updated 8 files:
  - packages/shadcn-ui/src/ui/sidebar.tsx
  - packages/shadcn-ui/src/ui/button.tsx
  - packages/shadcn-ui/src/ui/separator.tsx
  - packages/shadcn-ui/src/ui/sheet.tsx
  - packages/shadcn-ui/src/ui/tooltip.tsx
  - packages/shadcn-ui/src/ui/input.tsx
  - packages/shadcn-ui/src/hooks/use-mobile.tsx
  - packages/shadcn-ui/src/ui/skeleton.tsx
```

**Comment by bzp2010** [comment]
About `add` command:

```
➜  my-workspaces git:(test) ✗ TS_NODE_PROJECT=tsconfig.base.json pnpm dlx shadcn@latest add sidebar
✔ Checking registry.
✔ Updating packages/shadcn-ui-utils/src/tailwind.config.ts
✔ Updating packages/shadcn-ui-utils/src/global.css
✔ Installing dependencies.
✔ The file sidebar.tsx already exists. Would you like to overwrite? … yes
✔ The file button.tsx already exists. Would you like to overwrite? … yes
✔ The file separator.tsx already exists. Would you like to overwrite? … yes
✔ The file sheet.tsx already exists. Would you like to overwrite? … yes
✔ The file tooltip.tsx already exists. Would you like to overwrite? … yes
✔ The file input.tsx already exists. Would you like to overwrite? … yes
✔ The file use-mobile.tsx already exists. Would you like to overwrite? … yes
✔ The file skeleton.tsx already exists. Would you like to overwrite? … yes
ℹ Updated 8 files:
  - packages/shadcn-ui/src/ui/sidebar.tsx
  - packages/shadcn-ui/src/ui/button.tsx
  - packages/shadcn-ui/src/ui/separator.tsx
  - packages/shadcn-ui/src/ui/sheet.tsx
  - packages/shadcn-ui/src/ui/tooltip.tsx
  - packages/shadcn-ui/src/ui/input.tsx
  - packages/shadcn-ui/src/hooks/use-mobile.tsx
  - packages/shadcn-ui/src/ui/skeleton.tsx

➜  my-workspaces git:(test) ✗ TS_NODE_PROJECT=tsconfig.base.json pnpm dlx shadcn@latest add sidebar --overwrite
✔ Checking registry.
✔ Updating packages/shadcn-ui-utils/src/tailwind.config.ts
✔ Updating packages/shadcn-ui-utils/src/global.css
✔ Installing dependencies.
ℹ Updated 8 files:
  - packages/shadcn-ui/src/ui/sidebar.tsx
  - packages/shadcn-ui/src/ui/button.tsx
  - packages/shadcn-ui/src/ui/separator.tsx
  - packages/shadcn-ui/src/ui/sheet.tsx
  - packages/shadcn-ui/src/ui/tooltip.tsx
  - packages/shadcn-ui/src/ui/input.tsx
  - packages/shadcn-ui/src/hooks/use-mobile.tsx
  - packages/shadcn-ui/src/ui/skeleton.tsx
```

File: e2e/shadcn-ui-e2e/tests/shadcn-ui.spec.ts [hunk 1]
```diff
@@ -22,14 +22,15 @@ describe('shadcn/ui e2e', () => {
...
     )).not.toThrow()
-  })
 
-  // it('should be able add a button', async () => {
-  //   await runNxCommandAsync(`add ${uiLibName} button`)
-  //
-  //   expect(() => checkFilesExist(
-  //     `${uiLibName}/src/button.tsx`,
-  //   )).not.toThrow()
-  // })
```
Comment (TriPSs) [inline e2e/shadcn-ui-e2e/tests/shadcn-ui.spec.ts:33]: You think we can add this test? Prevents us from releasing a version again that suddenly stops working.

**Comment by bzp2010** [comment]
I have fixed the timeout issue, please help re-trigger the CI. 🙏

**Comment by TriPSs** [comment]
Thanks for the PR! 🎉

**Comment by bzp2010** [comment]
Tested with 4.2.0 in a project and `add-component` works fine. ✨

**Pull request merged**